client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/bootstrap/BootstrapManager.java

```diff
@@ -16,26 +16,26 @@
 
 package org.kaaproject.kaa.client.bootstrap;
 
-import java.util.List;
-
```

Comment:
I don't know, how I can avoid this. I think formatting help to read code faster and maintain t faster. I use only IDE formatting. I understand, that hard to read, but don't know how I can help in this. Don't want to point to smth, but maybe we can create some formatting rules, one configuration for all? if formatting in this project have some special requirements, we can format all projects to one style. It will be very helpful, as I can see this...